process/TI-Gives+Gets.md

```diff
@@ -0,0 +1,87 @@
+ * Projects must have defined a contributor guide, which makes it clear how and when contributors should be given increasing responsibilities towards maintainership of the project. (Example guides: Sigstore, AllStar)
+ * Projects should be able to show adoption by multiple parties and adoption's value to the open source community and/or end users (may include adoption of beta/early versions) with the intent to showcase wide adoption by the project's consumers.
+ * TI must have documented, initial group governance.
+ * Maintains a point of contact for vulnerability reports in the security.md
+ * Implements, practices, and refines mature software development and release practices such as following a version schema.
+ * TI Follows security best practices (as recommended by the OpenSSF and others), including passing the OpenSSF Best Practices criteria
+ * Project should be integrating with Scorecards
+ * Begins to establish the appropriate governance that enables its sustainment for potential graduation.
+ * Projects should be Securing Code Repository -> Managing Contributions Commit Signing , Secret Scanning, Code Scanning (OSFUZZ at a minimum) + Self-assessment Should OpenSSF require these if the SCM supports it, especially using Sigstore?
+
+### Gets/Benefits
+
+ TI eligible to receive all Gets from Sandbox plus:
+ * Receives infrastructure support
+ * Receives consideration as in-scope for any submission to an OpenSSF-managed conference or event.
+ * Project may request custom OpenSSF Logo for group
+ * Projects may use the OpenSSF logo to promote their project (in accordance with the trademark guidelines). Projects may not be referred to as an "OpenSSF Project" or "OpenSSF $ProjectName." Projects may say they are an "OpenSSF Incubating Project."
+ * With additional TAC or WG approval, may fundraise for dedicated project funds, coordinated by the OpenSSF.
+ * Receives support with vulnerability disclosure from the OpenSSF (Vulnerability Disclosure WG).
+ * May post project updates and tutorials to the OpenSSF blog.
+
+
+## Graduated level Gives & Gets
+
+### Gives/Requirements
+
+ All requirements of Incubating must be fulfilled and additionally:
+ * Projects must be able to show a consistent release cadence.
+ * Maintains a point of contact for vulnerability reports and follow coordinated vulnerability disclosure practices.
+ * Implements, practices, and refines mature software development and release practices, such as adherence to semantic versioning, and having a declared policy for stable releases and backported fixes.
+ * Projects must have documented project governance and be able to demonstrate that governance in action.
+ * When applicable, projects must have completed a security audit through a third party and addressed audit findings and recommendations.
+ *  When applicable, Projects should achieve **BLAH** level of SLSA
```

Comment:
SLSA Level 3?

Reply:
yeah, that's also what I had in mind but I am wondering whether we need to say SLSA Build Level 3 until more is defined.

Reply:
Like I commented above, very specific security requirements tend not to age well. As it stands, there are multiple interpretations of "prevent secret material used to sign the provenance from being accessible to the user-defined build steps" which is part of SLSA v1.0 Build Level 3.

... but I don't have a great suggestion here. Maybe leave this off? Or say something like "Projects should harden their build systems in accordance with the SLSA Framework"?